components/identity-extensions/backend-oauth-authenticator/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>identity-extensions</artifactId>
+        <groupId>org.wso2.carbon.devicemgt</groupId>
+        <version>0.9.2-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>bundle</packaging>
+    <name>WSO2 Carbon - OAuth Back End Authenticator </name>
+    <artifactId>org.wso2.carbon.identity.authenticator.backend.oauth</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.utils</artifactId>
+            <version>${carbon.kernel.version}</version>
```

Comment:
Remove all explicitly defined versions in each and every dependency configured in the dependencies section. These need to be inherited from the parent.